### PR TITLE
Fix issue #6

### PR DESCRIPTION
**Pull request by hassanabidpk**

### CODE REVIEW

**Comment by hassanabidpk** [comment]
I had to delete previous one and push again.

**Comment by hassanabidpk** [comment]
Did you check it ?

File: hacks/admin.py [hunk 1]
```diff
@@ -1,5 +1,12 @@
 from django.contrib import admin
 from .models import Hack, Category
+from django.template.defaultfilters import slugify
 
-admin.site.register(Hack)
-admin.site.register(Category)
+class CategoryAdmin(admin.ModelAdmin):
+    exclude = ('slug',)
```
Comment (jinpark) [inline hacks/admin.py]: I was thinking of making this a `read-only` but this is fine too :)

**Comment by jinpark** [comment]
Looks fine but I still see the `debug.log` thing. I can kill it in another pr

**Pull request merged**